Makefile

```diff
@@ -589,6 +589,7 @@ scalaclean:
 scalapkg:
 	(cd $(ROOTDIR)/scala-package; \
 		mvn package -P$(SCALA_PKG_PROFILE),$(SCALA_VERSION_PROFILE) -Dcxx="$(CXX)" \
+		    -Dbuild.platform="$(SCALA_PKG_PROFILE)" \
```

Comment:
Can't we used this param in the `profile` directly?

Reply:
I think I have tried this, I couldn't use read this from the profile in release/release-perform that is why I had to add an additional property.
```
-Darguments=""-Dbuild\.platform=\""$(SCALA_PKG_PROFILE)\""
```